prow/cmd/deck/static/dialog-polyfill.js

```diff
@@ -0,0 +1,738 @@
+(function() {
```

Comment:
license? I suspect this is https://github.com/GoogleChrome/dialog-polyfill which has [a BSD license](https://github.com/GoogleChrome/dialog-polyfill/blob/master/LICENSE) which requires redistributing the license.